### ISSUE TITLE
TemporaryJobs outputs to stdout unless run as junit test

### ISSUE THREAD

**Issue by mzeo** [spotify/helios#883]
Should at least use logger.

**Comment by mattnworb** [comment]
what does the slf4j setup look like in your project where this happens? Is `slf4j-simple` on the classpath by any chance? http://www.slf4j.org/apidocs/org/slf4j/impl/SimpleLogger.html

**Comment by mzeo** [comment]
I believe the problem comes from here:

https://github.com/spotify/helios/blob/master/helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java#L145

As I'm not using junit, the test writer isn't created (https://github.com/spotify/helios/blob/master/helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java#L337)

Otherwise my logback.xml works as I expect, and looks like this:
```
<configuration>
  <appender name="STDERR"
            class="ch.qos.logback.core.ConsoleAppender">
    <param name="target" value="System.err"/>
    <encoder>
      <pattern>
        %d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
      </pattern>
    </encoder>
  </appender>

  <root level="DEBUG">
    <appender-ref ref="STDERR" />
  </root>
</configuration>
```

**Comment by mattnworb** [comment]
fixed in #1022
